src/modules/portfolio/components/orders/filled-orders.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+// import PositionsMarketsList from "modules/portfolio/components/positions-markets-list/positions-markets-list";
+import FilterSwitchBox from "modules/portfolio/components/common/filter-switch-box";
+import MarketRow from "modules/portfolio/components/common/market-row";
+
+const sortByOptions = [
+  {
+    label: "View by Market",
+    value: "creationTime",
+    comp: null
+  },
+  {
+    label: "View by Outcome",
+    value: "endTime",
+    comp: null
+  }
+];
+
+export default class FilledOrders extends Component {
+  static propTypes = {
+    markets: PropTypes.array.isRequired,
+    filledOrders: PropTypes.array.isRequired,
+    loadAccountTrades: PropTypes.func.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      filteredData: props.markets,
+      viewByMarkets: true
+    };
+
+    this.updateFilteredData = this.updateFilteredData.bind(this);
+    this.filterComp = this.filterComp.bind(this);
+    this.switchView = this.switchView.bind(this);
+  }
+
+  componentWillMount() {
+    const { loadAccountTrades } = this.props;
+    loadAccountTrades();
+  }
+
+  updateFilteredData(filteredData) {
```

Comment:
all these methods could be arrow functions then you don't have to bind to `this`